tests/unit/exchanges/ethfinexOrder.test.js

```diff
@@ -161,7 +161,12 @@ test('Previously made ethfinex order cancelled and not takeable anymore', async
 });
 
 test('Withdraw (unwrap) maker asset of cancelled order', async () => {
-  increaseTime(25*60*60);
+  // skip past order expiration
+  const depositLockEnd = await zrxWrapperLock.methods.depositLock(
+    trading.options.address
+  ).call();
+  const latestBlockTimestamp = (await web3.eth.getBlock()).timestamp;
+  await increaseTime(depositLockEnd - latestBlockTimestamp + 1);
```

Comment:
Good catch! It is definitely best to be intentional with the time intervals we set. It's tough to debug when there are no comments about why we're waiting a particular amount of time.